### ISSUE TITLE
Linux support?

### ISSUE THREAD

**Issue by nvhai245** [wabbajack-tools/wabbajack#2521]

**Comment by EzioTheDeadPoet** [comment]
It was tried.
Didn't work out.
There is an unnofficial channel on the discord.
https://discord.gg/wabbajack

**Comment by this-is-just-con-tribute** [comment]
> It was tried. Didn't work out. There is an unnofficial channel on the discord. https://discord.gg/wabbajack

come on, tell me more!

**Comment by EzioTheDeadPoet** [comment]
> > It was tried. Didn't work out. There is an unnofficial channel on the discord. https://discord.gg/wabbajack
> 
> come on, tell me more!


Achieving native Linux support would require a complete rework of how the app functions and to account for dozens of edge cases how games are installed on Linux.
Not to mention that Mod Organizer 2 also doesn't have native Linux support so we would need to implement a new way for installing that.

This is a community project and nobody that is currently contributing to the app has the time to work on that big of an overhaul for a way too small user base.

That said we are open to contributions from community members that want to do that work in their free time.

But until someone volunteers to rewrite basically the whole app to account for all the Linux quirks while running on Linux while not breaking windows functionality there won't be in any official Linux support.


There is a help channel on the discord where other users share there results of messing with Proton, Bottles, Wine and the likes for using WJ.
If you really want to use Wabbajack on Linux go there.

To recap:
- No official support planned
- There are workarounds shared on the discord

**Comment by this-is-just-con-tribute** [comment]
> > > It was tried. Didn't work out. There is an unnofficial channel on the discord. https://discord.gg/wabbajack
> > 
> > 
> > come on, tell me more!
> 
> Achieving native Linux support would require a complete rework of how the app functions and to account for dozens of edge cases how games are installed on Linux. Not to mention that Mod Organizer 2 also doesn't have native Linux support so we would need to implement a new way for installing that.
> 
> This is a community project and nobody that is currently contributing to the app has the time to work on that big of an overhaul for a way too small user base.
> 
> That said we are open to contributions from community members that want to do that work in their free time.
> 
> But until someone volunteers to rewrite basically the whole app to account for all the Linux quirks while running on Linux while not breaking windows functionality there won't be in any official Linux support.
> 
> There is a help channel on the discord where other users share there results of messing with Proton, Bottles, Wine and the likes for using WJ. If you really want to use Wabbajack on Linux go there.
> 
> To recap:
> 
>     * No official support planned
> 
>     * There are workarounds shared on the discord

dang it. well, since modding is kind of niche, I guess that's fair. Hopefully as linux market share grows, you guys will either change your mind or someone will make a wabbajack-list compatible  thingy for linux.

**Comment by EzioTheDeadPoet** [comment]
The guides that are there seem to work pretty reliably for the people that tried them but it requires manual steps to work and it is just not something we can officially support.

**Comment by this-is-just-con-tribute** [comment]
> The guides that are there seem to work pretty reliably for the people that tried them but it requires manual steps to work and it is just not something we can officially support.

Some figured out a script to automate everything, but I'm pretty sure it doesn't work on steam deck, idk if it's even possible to automate it on steam deck.

the script in question: https://github.com/Omni-guides/Wabbajack-Modlist-Linux/blob/main/binaries/WabbajackWine.sh ok admittedly it only works for steam rn but it may be possible to make it work with gog, otherwise you just reinstall the gog games and it should work (so I'm told, I will test it later, but I've already procrastinated enough today.)
